test/oas3/execute/main.js

```diff
@@ -782,7 +782,7 @@ describe('buildRequest - OpenAPI Specification 3.0', () => {
 
       expect(req).toEqual({
         method: 'POST',
-        url: `/?arrayOfObjects=${escape('{"a":{"b":"c"}}')},${escape('{"d":{"e":"f"}}')}&arrayOfArrays=${escape('[{"a":{"b":"c"}}]')}`,
+        url: `/?arrayOfObjects=${escape('{"a":{"b":"c"}},{"d":{"e":"f"}}')}&arrayOfArrays=${escape('[{"a":{"b":"c"}}]')}`,
```

Comment:
Let's use string literals so that we see what we're comparing here. Like in other tests you did in this PR.